tests/particle_interpolator_bilinear_add_particles.prm

```diff
@@ -0,0 +1,128 @@
+# A test based on the SolKz benchmark that performs bilinear
```

Comment:
Please update this test description to explain what this test does now. In particular that it tests to interpolate multiple properties with the bilinear least squares interpolator.